src/AutoRest.CSharp.V3/AutoRest/Communication/IPluginCommunication.cs

```diff
@@ -3,15 +3,15 @@
 
 using System.Threading.Tasks;
 using AutoRest.CSharp.V3.AutoRest.Communication.Serialization.Models;
+using AutoRest.CSharp.V3.AutoRest.Plugins;
 
 namespace AutoRest.CSharp.V3.AutoRest.Communication
 {
     internal interface IPluginCommunication
     {
         string PluginName { get; }
-        Task<string> ReadFile(string filename);
-        Task<T> GetValue<T>(string key);
-        Task<string[]> ListInputs(string? artifactType = null);
```

Comment:
So, these were AutoRest communication concepts (meaning, the JsonRPC API for AutoRest). So, changing them to non-AutoRest related methods would make us not have plugin communication (multiple plugins) in the future if we wanted/needed it.

The interface here was only so other plugin communcation methods (such as our Host communication) could communicate in the same way any other AutoRest plugin would.

Reply:
Yeah, you are right, I'll redo this change without messing up this abstraciton.